sdp_ompl_ros/PlanningEnvironment space
sdp_ompl_ros/PlanningProblem problem
---
sdp_ompl_ros/Path plan

